### PR TITLE
Hotfix/aut 2869/responsive background graphic interactions

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-2869
https://oat-sa.atlassian.net/browse/INV-624

Backport of https://github.com/oat-sa/tao-item-runner-qti-fe/pull/351 to November Release [taoQtiItem 29.14.4.6](https://github.com/oat-sa/tao-community/blob/2022.11.11/composer.json#L39) 

To DO: 

- [x] release https://github.com/oat-sa/tao-item-runner-qti-fe/pull/353
- [x] update package.json and package-lock.json
- [x] bundle js files

### CODE REVIEW

File: views/js/qtiCreator/widgets/interactions/helpers/bgImage.js [hunk 1]
```diff
@@ -13,84 +13,32 @@
...
-    /**
-     * @exports
-     */
-
-    /**
-     * Handle images that are larger than the canvas
-     *
-     * @param widget
-     */
-    function handleOversizedImages(widget) {
```
Comment (Karol-Stelmaczonek) [inline views/js/qtiCreator/widgets/interactions/helpers/bgImage.js:40]: This function wasn't used anywhere that's right? I checked in code base but better ask if this removal was intentional :)

Comment (bziondik) [reply views/js/qtiCreator/widgets/interactions/helpers/bgImage.js:40]: it is inner function, that no need now

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by marpesia — APPROVED** [review]
LGTM

**Pull request merged**